src/hooks/useLoadAccountData.js

```diff
@@ -27,7 +28,8 @@ export default function useLoadAccountData() {
       if (network === networkTypes.mainnet) {
         const p1 = dispatch(dataLoadState());
         const p2 = dispatch(uniqueTokensLoadState());
-        promises.push(p1, p2);
+        const p3 = dispatch(transactionSignaturesLoadState());
```

Comment:
great! could we just move these changes to `useLoadGlobalData` instead of `useLoadAccountData` (this one is for "account specific" redux state we load)